### DISCUSSION TITLE
Milton Keynes Cities are missing

### DISCUSSION THREAD

**Discussion by vnooli** [dr5hn/countries-states-cities-database#623] [Q&A]
Hello @dr5hn,
Recently I've being using API for country state and city details.
In the process found that for **Milton Keynes** state in **United Kingdom** country **cities** are missing, and for few others as well missing.
Can you please update the api to include missing states and cities ASAP, As we are going for client demos.
I can provide missing list as well if you wish to know.

Thanks & regards in advance,
Nooli Vinod Kumar Patel.

**Comment by nigelhorne** [comment]
Milton Keynes is not a state, it's a town.
